draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+
+For a total of :math:`m` actions in a transfer, the prover can still sign the `SIGHASH` of the transaction using the binding signature key
+
+:math:`\mathsf{bsk} = \sum_{\mathsf{ \forall i\in \{1,...,m\}}} \mathsf{rcv_{i}}`
+
+Then we have that the verifier computes
+
+:math:`\mathsf{bvk = (\sum cv_i^{net})}  - \mathsf{ ValueCommit_0^{Orchard}(v^{balanceOrchard})} = \sum \mathsf{rcv_{i}^{net}}\mathcal{R}^{\mathsf{Orchard}}`
+
+
+And uses it to verify the binding signature, as described in §4.14 of the Zcash Specification [#protocol-binding]_, by verifying the `bindingSignature` on the `SIGHASH` message.
+
+As in the Orchard protocol, the binding signature verification key, :math:`\mathsf{bvk}`, will only be valid (and hence verify the signature correctly, as long as all the value commitments (and corresponding value balances) are equal to zero. In contrast, in this protocol, the value commitments only cancel out **per asset type**, as the Pedersen commitments add up homomorphically only with respect to the same value base point.
+
+Split Notes
+-----------
+
+One of the key functionalities in a UTXO based protocol is the fact that input notes are usually split in two (or more) output notes, as in most cases, not all the value in a single note is sent to a single output. This is called a 1-to-many (Orchard) transaction. In order to cope with this today, the input note of the second (third and more) Action (which we call split notes and split Actions respectively) is a *dummy spend note* [#protocol-dummynotes]_. Basically, the input note is “faked” inside of the proof in order to hide which action contains the *real* spend note.
+
+This, however, brings some issues when it comes to adding multiple asset types, as the output note of the split Actions *cannot* be of *any* asset type, it must be enforced to be an actual output of a GroupHash computation (in fact we want it to be of the same type as the original input note, but the binding signature takes care that the proper balancing is performed). If not, then the prover could essentially input a multiple (or linear combination of) an existing type, with the goal to attack the network by overflowing the ZEC value balance and hence counterfeiting ZEC funds.
+
+In order to prevent this, we make some modifications to the circuit. Specifically we remove the dummy note functionality for custom assets and we enforce that *every* input note to an ZSA Action must be proven to exist in the set of note commitments in the Merkle Tree. We then enforce this real note to be “unspendable” in the sense that its value
```

Comment:
```suggestion
In order to prevent this, we make some modifications to the circuit. Specifically we remove the dummy note functionality for custom Assets and we enforce that *every* input note to an ZSA Action must be proven to exist in the set of note commitments in the Merkle Tree. We then enforce this real note to be “unspendable” in the sense that its value
```

Reply:
Added these changes in [PR#649](https://github.com/zcash/zips/pull/649).